cmake-modules/AddTestCMocka.cmake

```diff
@@ -65,6 +65,11 @@ function(ADD_CMOCKA_TEST _TARGET_NAME)
 
     add_executable(${_TARGET_NAME} ${_add_cmocka_test_SOURCES})
 
+    # Suppress clobber warning for longjmp
+    if(CMAKE_C_COMPILER_ID MATCHES "GNU")
+      target_compile_options(${_TARGET_NAME} PRIVATE -Wno-clobbered)
```

Comment:
What does applying the `PRIVATE` keyword do in this case?

Reply:
With executables I don't think it really matters since it won't be consumed by anything else. But in theory with CMake you can make this public or private which makes them transitive (or not) if another lib/executable consumes them. Since this executable is the end of the road I'm almost positive it doesn't matter.